### ISSUE TITLE
Updating Fig with brew update causes issues

### ISSUE THREAD

**Issue by DarthGigi** [withfig/fig#2032]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

macOS 13.0.0 (22A380)

### Expected behaviour

To brew to update fig without any errors.

### Actual behaviour

When I run `brew update` to update all my casks (including fig) this is the output:

```
==> Upgrading 1 outdated packages:
fig 2.7.5 -> 2.7.7
==> Upgrading fig
==> Caveats
Please launch the Fig application to finish setup...

==> Downloading https://repo.fig.io/generic/stable/asset/2.7.7/universal/fig.dmg
######################################################################## 100.0%
==> Removing launchctl service io.fig.launcher
Password:
==> Removing launchctl service io.fig.uninstall
==> Removing launchctl service io.fig.dotfiles-daemon
==> Quitting application 'io.fig.cursor'...
Application 'io.fig.cursor' quit successfully.
==> Running uninstall script /Applications/Fig.app/Contents/MacOS/fig-darwin-uni
==> Purging files for version 2.7.7 of Cask fig
Error: fig: It seems the App source '/Applications/Fig.app' is not there.
```

I will have to use `brew reinstall --cask fig` to update/reinstall fig.

### Steps to reproduce

Happens every time I use brew to update fig

### Environment

```yaml
fig-details:
  - 2.7.7
hardware-info:
  - model: 
  - model-id: 
  - chip-id: Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
  - cores: 6
  - mem: 16.00 GB
os-info:
  - macOS 13.0.0 (22A380)
environment:
  - shell: /bin/zsh
  - terminal: iterm
  - cwd: /Users/gigi
  - exe-path: /usr/local/bin/fig
  - install-method: brew
  - env-vars:
    - FIGTERM_SESSION_ID: b50ff424-4ead-42ed-97e3-be0cb5997468
    - FIG_HOSTNAME: Gigis-MacBook-Pro.local
    - FIG_TERM: 2.7.7
    - PATH: /usr/local/bin:/usr/local/sbin:/usr/local/opt/make/libexec/gnubin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Applications/Little Snitch.app/Contents/Components:/Library/Apple/usr/bin:/Users/gigi/.fig/bin:/Users/gigi/.local/bin
    - SHELL: /bin/zsh
    - TERM: xterm-256color
    - FIG_PID: 6405
    - FIG_PARENT: b50ff424-4ead-42ed-97e3-be0cb5997468
    - FIG_SET_PARENT: b50ff424-4ead-42ed-97e3-be0cb5997468
```

**Comment by DarthGigi** [comment]
Oh I just noticed that this is a duplicate of https://github.com/withfig/fig/issues/1985

**Comment by grant0417** [comment]
Going to close since I think this is resolved and is a dupe of #1985
